### ISSUE TITLE
Dangerous address when interacting with your bridge

### ISSUE THREAD

**Issue by luislucena16** [kroma-network/kroma#264]
### Issue type

Bug

### Kroma version

v1.3.2

### Current behavior?

there is an alert when interacting with your bridge, which indicates that it was flagged as `phishing`

### Expected Behavior?

I hope not to interact with possible risky addresses!

### Standalone code or description to reproduce the issue

![pishing](https://github.com/kroma-network/kroma/assets/86749595/780d4f7e-42ad-4e3e-8998-05beb64daabb)

### Additional context

this happens when trying to change tokens from ethereum sepolia to kroma sepolia, after a long time searching I managed to get the bridge...!

**Comment by Pangssu** [comment]
Thank you for reporting this.
What is the URL of the bridge you used? The URL of the official bridge is https://hub.kroma.network/bridge. 
I checked with Blockfence, but no issues were reported.

**Comment by luislucena16** [comment]
oh, the one I used was this one: https://kroma.network/bridge
I got it from their [official documentation](https://docs.kroma.network/user-guide/use-the-bridge), should update the documentation asap

**Comment by Pangssu** [comment]
That's an official bridge too. I think someone made a false report.

**Comment by luislucena16** [comment]
really? well, maybe it happened as you mention! the solution is with blockfence!

**Comment by Pangssu** [comment]
I'll have our team look into this, thank you for letting us know.

**Comment by luislucena16** [comment]
thank you for your quick response and solution!

**Comment by luislucena16** [comment]
is there an easier way to get tokens from Kroma Sepolia? don't they have their own faucet or do they send them internally? I'm needing some to test certain things for my contracts

**Comment by Pangssu** [comment]
Do you mean "sepETH" by the token you're talking about? If so, sorry, we don't have enogh sepETH too :'(

**Comment by luislucena16** [comment]
yes, i'm doing some interesting things in Kroma and i found it difficult to interact with their network, until i got the bridge to send the sepETH from ethereum sepolia to kroma sepolia! i'll keep on with what i have

**Comment by Pangssu** [comment]
Cool, thanks.
And I will close this issue once that issue is resolved.

**Comment by luislucena16** [comment]
great! thank you!

**Comment by luislucena16** [comment]
hey @Pangssu because when I go to apply for sepolia here:
https://www.l2faucet.com/kroma
it sends me to scan a QR or asks me for an extension, it is invasive

**Comment by Pangssu** [comment]
You might think so. The reason L2Faucet requires a QR code scan seems to be device authentication to prevent abusing. 
See the article below.
https://docs.ata.network/product/l2faucet

**Comment by luislucena16** [comment]
thanks for the article! but IMHO I think it's better to do it with social login if that's what you're looking for, I'm sure very few people would be willing to scan the QR

**Comment by 0xHansLee** [comment]
This has been resolved. There is no caution of phishing for our official bridge web. So I close this issue.
